pkgs/top-level/all-packages.nix

```diff
@@ -33315,6 +33315,8 @@ with pkgs;
     boost = boost175;
   };
 
+  libtas = callPackage ../applications/misc/libtas { };
```

Comment:
Please move the package to `pkgs/by-name/li/libtas/package.nix` and drop this line, thanks.